doc/integrator/docker.rst

```diff
@@ -231,3 +231,21 @@ add something like that in your composition:
 
 When you do an upgrade, backup your DB and upgrade this container first. It will update your
 DB schema, if needed.
+
+Clean
+-----
+
+Docker don't clean anything automatically, and espacially the images take many place.
+You can use the following commands to do a maual clean:
```

Comment:
Suggestion:
You can use the following commands to manually remove docker files.